renderdoc/driver/dxgi/dxgi_common.cpp

```diff
@@ -1391,7 +1391,8 @@ DXGI_FORMAT MakeDXGIFormat(ResourceFormat fmt)
         break;
       case eSpecial_S8: RDCERR("D3D has no stencil-only format"); break;
       case eSpecial_D16S8: RDCERR("D3D has no D16S8 format"); break;
-      default: RDCERR("Unrecognised/unsupported special format %u", fmt.specialFormat); break;
+      default:
+        break;    // Return DXGI_FORMAT_UNKNOWN to convert texture.
```

Comment:
Done. Obviously you still want to keep the RDCERR for unrecognized component counts/widths that are just strange. 

About eSpecial_YUV, do you still want that remapped? You have an RDCERR but also a DXGI format.